app.py

```diff
@@ -4,18 +4,24 @@
 from flask import Flask, Response, render_template
 app = Flask(__name__)
 
+
 config_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), "configs", "board.json")
 
 
-@app.route('/')
+@app.route('/board')
 def main():
     return render_template("board.html", development=True)
 
 
+# TODO: Move to an api endpoint
 @app.route('/map')
 def get_map():
     map = Map.load_from_config(config_path, game_type="3p_4p_default")
 
     return Response(response=map.to_json(),
                     status=200,
                     mimetype="application/json")
+
+
+# API
+exec(open("gaia/api.py").read())
```

Comment:
Could we change this to use Python's import system instead?

Reply:
Try `import gaia.api`

Reply:
hmm im struggling to get that to work. Will take another look tomorrow

Reply:
So all the tutorials for flask restplus use either the Flask app.route or the Flask_restplus api.route, but not both. Why don't we choose one and have all of those endpoints live in the main file (and delete the other file).

Reply:
Ya Im cool with that. Lets namespace the api though and make a clear separation between view and api within app.py. Api routes should be in the format api/v1/moves going forward

Reply:
Lets use app.route if they are incompatible.

Reply:
The motivation behind the separation was flask's bare bones functionality will probably work for our views, but it might be nice to have the added documentation for our api. We don't need flask rest plus for our views IDT.